speech_transcribe_async.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("LongRunningPromise",  "speech_transcribe_async")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-speech
+
+# sample-metadata
+#   title: Transcribe Audio File using Long Running Operation (Local File) (LRO)
+#   description: Transcribe a long audio file using asynchronous speech recognition
+#   usage: python3 samples/v1/speech_transcribe_async.py [--local_file_path "resources/brooklyn_bridge.raw"]
+
+# [START speech_transcribe_async]
+from google.cloud import speech_v1
+from google.cloud.speech_v1 import enums
```

Comment:
From the Authoring Guide:

```
Beyond PEP8, there are several idioms and style nits we prefer.

from gcloud import datastore 

instead of 

from gcloud.datastore import Client
```

But use your best judgement here, I am not too picky on this one.

Reply:
I think enums is a module, so this does match the Authoring Guide.